x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/queries/SemanticQueryBuilderTests.java

```diff
@@ -96,7 +96,10 @@ public static void setInferenceResultType() {
         // These are class variables because they are used when initializing additional mappings, which happens once per test suite run in
         // AbstractBuilderTestCase#beforeTest as part of service holder creation.
         inferenceResultType = randomFrom(InferenceResultType.values());
-        denseVectorElementType = randomFrom(DenseVectorFieldMapper.ElementType.values());
+        denseVectorElementType = randomValueOtherThan(
+            DenseVectorFieldMapper.ElementType.BIT,
+            () -> randomFrom(DenseVectorFieldMapper.ElementType.values())
+        ); // TODO: Support bit elements once KNN bit vector queries are available
```

Comment:
The `semantic` query cannot support the `BIT` element type until a corresponding KNN query is available